<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>std::vector demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="list_widget"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QLineEdit" name="txt_elem_content"/>
         </item>
         <item>
          <widget class="QPushButton" name="btn_edit">
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_find">
           <property name="text">
            <string>find</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLineEdit" name="le_count"/>
         </item>
         <item>
          <widget class="QPushButton" name="btn_count">
           <property name="text">
            <string>count</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_count">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="lbl_metod">
         <property name="text">
          <string>Методы</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="5" column="0">
          <widget class="QPushButton" name="btn_min_element">
           <property name="text">
            <string>min_element</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="btn_erase">
           <property name="text">
            <string>erase</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="btn_insert">
           <property name="text">
            <string>insert</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QPushButton" name="btn_left">
           <property name="text">
            <string>--</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QPushButton" name="btn_sort_reg">
           <property name="text">
            <string>merge sOrT</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lbl_algorithm">
           <property name="text">
            <string>Алгоритмы</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QPushButton" name="btn_right">
           <property name="text">
            <string>++</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QPushButton" name="btn_sort">
           <property name="text">
            <string>merge sort</string>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QPushButton" name="btn_begin">
           <property name="text">
            <string>begin</string>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QPushButton" name="btn_end">
           <property name="text">
            <string>end</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="btn_clear">
           <property name="text">
            <string>clear</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QPushButton" name="btn_shuffle">
           <property name="text">
            <string>shuffle</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QPushButton" name="btn_unique">
           <property name="text">
            <string>unique</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QPushButton" name="btn_reverse">
           <property name="text">
            <string>reverse</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btn_push_front">
           <property name="text">
            <string>push_front</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="lbl_iterator">
           <property name="text">
            <string>Итератор</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="btn_pop_back">
           <property name="text">
            <string>pop_back</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="btn_pop_front">
           <property name="text">
            <string>pop_front</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="btn_push_back">
           <property name="text">
            <string>push_back</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="btn_max_element">
           <property name="text">
            <string>max_element</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QPushButton" name="btn_lower_bound">
           <property name="text">
            <string>lower_bound</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QPushButton" name="btn_upper_bound">
           <property name="text">
            <string>upper_bound</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="lbl">
         <property name="text">
          <string>Загатовки</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btn_tea">
           <property name="text">
            <string>= чай</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_cakes">
           <property name="text">
            <string>= торты</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lbl_size">
           <property name="text">
            <string>size() :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txt_size"/>
         </item>
         <item>
          <widget class="QPushButton" name="btn_resize">
           <property name="text">
            <string>resize</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>